pipelines/h3c/rtp/v1/task_scripts/do_ADD_MC_ANT_METRICS.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+# import common functions
+src_dir="$(dirname "$0")"
+source ${src_dir}/_common.sh
+
+# Parameters are set in the configuration file. Here we define their positions,
+# which must be consisten wtih the config.
+# 1 - (raw) filename
+fn=$(inject_hh ${1})
+
+# get ant_metrics filename
+metrics_f="${fn%.*}".ant_metrics.uvh5
```

Comment:
this `%.*` is neat, I have never seen it before but are we sure we want to blindly strip the last extension off? I presume we will only be sending uvh5 files through? and if this extension stripping is something that will be used often do we want to add it into the `_common.sh`?

Reply:
I think blindly stripping the extension is fine because that's how the output file in ant_metrics gets written (a call to strip_extension).